locustempus/main/views.py

```diff
@@ -671,9 +671,16 @@ def get(self, request, *args, **kwargs) -> HttpResponse:
 
 
 class ProjectCreateView(LoggedInFacultyMixin, View):
+
     def post(self, request, *args, **kwargs):
         course = get_object_or_404(Course, pk=kwargs.get('pk', None))
-        project = Project.objects.create(course=course)
+
+        if hasattr(settings, 'DEFAULT_BASE_MAP'):
```

Comment:
This might be more of a nit, but I think it might make more sense to also have a `DEFAULT_BASE_MAP` in settings_shared as well. It could be confusing to call it default if it only applies to testing settings. It would remove the need for a conditional check here, too.

Reply:
It's a good point. I thought of taking that approach...the only fussy bit is that the Project `base_map` attribute has the `mapbox` attribute hard-coded. I don't think a model default can be a setting. I can try though.

Reply:
@nbuonin - check commit 140c986 for the update.